zeek/bin/spicyz.cc

```diff
@@ -0,0 +1,226 @@
+                                              {"help", no_argument, nullptr, 'h'},
+                                              {"keep-tmps", no_argument, nullptr, 'T'},
+                                              {"library-path", required_argument, nullptr, 'L'},
+                                              {"optimize", no_argument, nullptr, 'O'},
+                                              {"output", required_argument, nullptr, 'o'},
+                                              {"output-c++", required_argument, nullptr, 'c'},
+                                              {"report-times", required_argument, nullptr, 'R'},
+                                              {"version", no_argument, nullptr, 'v'},
+                                              {nullptr, 0, nullptr, 0}};
+
+static void usage() {
+    std::cerr << "Usage: spicyz [options] <inputs>\n"
+                 "\n"
+                 "  -c | --output-c++ <prefix>      Print out all generated C++ code into files named with <prefix>.\n"
+                 "  -d | --debug                    Include debug instrumentation into generated code.\n"
+                 "  -o | --output-to <path>         Path for saving output.\n"
+                 "  -v | --version                  Print version information.\n"
+                 "  -A | --abort-on-exceptions      When executing compiled code, abort() instead of throwing HILTI "
+                 "exceptions.\n"
+                 "  -B | --show-backtraces          Include backtraces when reporting unhandled exceptions.\n"
+                 "  -C | --dump-code                Dump all generated code to disk for debugging.\n"
+                 "  -D | --compiler-debug <streams> Activate compile-time debugging output for given debug streams "
+                 "(comma-separated; 'help' for list).\n"
+                 "  -L | --library-path <path>      Add path to list of directories to search when importing modules.\n"
+                 "  -O | --optimize                 Build optimized release version of generated code.\n"
+                 "  -R | --report-times             Report a break-down of compiler's execution time.\n"
+                 "  -T | --keep-tmps                Do not delete any temporary files created.\n"
+                 "  -X | --debug-addl <addl>        Implies -d and adds selected additional instrumentation "
+                 "(comma-separated; see 'help' for list).\n"
+                 "\n"
+                 "Inputs can be *.spicy, *.evt, *.hlt, .cc/.cxx\n"
+                 "\n";
+}
+
+using hilti::Nothing;
+
+static hilti::Result<Nothing> parseOptions(int argc, char** argv, hilti::driver::Options* driver_options,
+                                           hilti::Options* compiler_options) {
+    while ( true ) {
+        int c = getopt_long(argc, argv, "ABc:CdX:D:L:o:ORTvh", long_driver_options, nullptr);
+
+        if ( c < 0 )
```

Comment:
Let's be more explicit.
```suggestion
        if ( c == -1 )
```